### PR TITLE
Packs P1 docs

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR updates the packs documentation and addresses the missing P1 documentation. See the Slack [thread](https://spectrocloud.slack.com/archives/CTZ6394G2/p1670961607892099) for more details 🧵

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1036#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:145]:
```suggestion
Due to security reasons, controller-manager, kube-schedule, and etcd run in the localhost causing Prometheus to fail when attempting to scrape metrics. Therefore, these targets are indicated as being down in Prometheus. Change the following in the Kubernetes pack layer in the cluster profile to scrape metrics for these services.

```

File: content/docs/06-integrations/00-vsphere-csi.md
Comment (ritawatson) [inline content/docs/06-integrations/00-vsphere-csi.md:18]:
```suggestion
The main goal of vSphere Container Storage Interface (CSI) is to expose vSphere storage and features to Kubernetes users. It offers an in-tree volume driver that has been actively used on various Kubernetes platforms by service providers, including on-prem. Cloud Native Storage (CNS) is a result of evolution and productization of vSphere Storage for Kubernetes and is also enterprise ready. 

```

**Review by ritawatson — APPROVED** [review]
LGTM with a couple minor changes. 

I'm not sure about my suggestion on line 18 - your call.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1036#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1036#pullrequestreview-1218065857" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
